src/components/ec-summary-info/ec-summary-info.spec.js

```diff
@@ -0,0 +1,191 @@
+import { mount } from '@vue/test-utils';
+
+import EcSummaryInfo from './ec-summary-info.vue';
+
+describe('EcSummaryInfo', () => {
+  function mountSummaryInfo(props, mountOpts) {
+    return mount(EcSummaryInfo, {
+      props: {
+        ...props,
+      },
+      ...mountOpts,
+    });
+  }
+
+  describe('icon', () => {
+    it('should render with the given "icon" prop', () => {
+      const wrapper = mountSummaryInfo({ iconName: 'simple-sell' });
+      expect(wrapper.findByDataTest('ec-summary-info__main-icon').exists()).toBe(true);
+      expect(wrapper.findByDataTest('ec-summary-info__main-icon__simple-sell').exists()).toBe(true);
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should not render with the not given "icon" prop', () => {
+      const wrapper = mountSummaryInfo();
+      expect(wrapper.findByDataTest('ec-summary-info__main-icon').exists()).toBe(false);
+    });
+  });
+
+  describe('lineItems', () => {
+    it('should render with the given "lineItems" prop', () => {
+      const wrapper = mountSummaryInfo({
+        lineItems: [
+          {
+            stylePreset: 'title',
```

Comment:
Let's do this stylePreset optional, and by default it will be 
`label` for the 0 line, 
`text` for 1 and 
'description' for 2
and 'text' that lines after 2

and it is better to name that stylePresets as here https://www.figma.com/file/qLv2in1OHCrSOEoNivIQA1/Guidelines?node-id=519%3A21849&t=EXGa1bcEESWoW1DH-4

Reply:
ok corrected